pysatNASA/instruments/timed_saber.py

```diff
@@ -38,20 +38,20 @@
 
 Warnings
 --------
-- Note on Temperature Errors: http://saber.gats-inc.com/temp_errors.php
+- Note on Temperature Errors: https://saber.gats-inc.com/temp_errors.php
```

Comment:
Is this a note or a warning?  Description should be consistent with the category.  Looks like this belongs in the Note section.

Reply:
It should probably go into Note since it doesn't raise a warning.